tests/integration/examples/test_mlflow_deployment.py

```diff
@@ -0,0 +1,38 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import pytest
+
+from tests.integration.examples.utils import run_example
+from zenml.post_execution.pipeline import get_pipeline
+
+
+def test_example(request: pytest.FixtureRequest) -> None:
+    """Runs the mlflow_deployment example."""
+
+    with run_example(
+        request=request,
+        name="mlflow_deployment",
```

Comment:
The mlflow deployment example starts a local daemon service to serve the model. At the end of the test, the daemon will become a zombie, destined to haunt us in the forever after. Can you do something about it ?

Reply:
Absolutely, I added a `service.stop()` call in case the service is still running for both the quickstart and the mlflow deployment example